### PR TITLE
Gravity top add-on

### PR DESCRIPTION
**Pull request by AlexBalo**
- Introduced custom gravity enum for dialogPlus to make it more clear which are the available options
- Handle default animation for gravity top

### CODE REVIEW

File: dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java [hunk 1]
```diff
@@ -211,15 +233,30 @@ private void initViews() {
...
+                ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT, convertedGravity
         );
         contentView.setLayoutParams(params);
         contentContainer.addView(contentView);
     }
 
     /**
+     * Convert DialogPlusGravity with content layout readable gravity
+     */
+    private int getLayoutGravity() {
```
Comment (orhanobut) [inline dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java]: I think we can just say getGravity(), what do you think?

Comment (AlexBalo) [inline dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java]: Yes, sure I agree and will change like that

File: dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java [hunk 2]
```diff
@@ -360,11 +402,11 @@ public boolean onTouch(View v, MotionEvent event) {
         private boolean isCancelable = true;
         private Holder holder;
         private int backgroundColorResourceId = INVALID;
-        private int gravity = Gravity.BOTTOM;
+        private Gravity gravity = Gravity.BOTTOM;
         private ScreenType screenType = ScreenType.HALF;
         private OnItemClickListener onItemClickListener;
-        private int inAnimation = R.anim.slide_in;
-        private int outAnimation = R.anim.slide_out;
+        private int inAnimation = -1;
```
Comment (orhanobut) [inline dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java]: magic number :) should we use something like INVALID or NO_ANIMATION?

**Comment by orhanobut** [comment]
finished reviewing

File: dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java (hunk 2 above)
Comment (AlexBalo) [inline dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java]: Yes, I will declare a variable like that :) Next time I will try to put more fantasy with magic numbers like 989 :D

Comment (orhanobut) [inline dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java]: haha :D it will be really magic number then :)

**Comment by AlexBalo** [comment]
Finished and pushed

**Pull request merged**